.bundlemonrc_template

```diff
@@ -0,0 +1,31 @@
+{
+    "baseDir": "./build",
+    "files": [
+        {
+            "path": "./$npm_package_name.zip",
```

Comment:
> If the ZIP file size is not monitored, this change can be discarded

Unless it's something you already agreed with @adrianq, I think we don't have gain much from checking both the ZIP and the contents zipped. I'd check just the contents. As a bonus, we could remove all the gymnastics required by the zip being on folder build/.

Reply:
I included both at the moment of creating the PR to show the options.
Adrian mentioned that the group method was fine. I will modify the PR accordingly.